content/renderer/media/media_permission_dispatcher.cc

```diff
@@ -142,15 +142,16 @@ void MediaPermissionDispatcher::OnPermissionStatus(
     blink::mojom::PermissionStatus status) {
   DVLOG(2) << __func__ << ": (" << request_id << ", " << status << ")";
   DCHECK(task_runner_->RunsTasksInCurrentSequence());
-
   auto iter = requests_.find(request_id);
   DCHECK(iter != requests_.end()) << "Request not found.";
 
   PermissionStatusCB permission_status_cb = std::move(iter->second);
   requests_.erase(iter);
 
-  std::move(permission_status_cb)
-      .Run(status == blink::mojom::PermissionStatus::GRANTED);
+  std::move(permission_status_cb).Run(true);
```

Comment:
Should we gate this by `USE_STARBOARD_MEDIA`?

This this always `true` for StarboardCdm?

Reply:
ah, that works, yeah!